viadot/sources/redshift_spectrum.py

```diff
@@ -59,36 +59,32 @@ def session(self) -> boto3.session.Session:
 
     def ls(
         self,
-        database: str = None,
+        database: str,
         name_contains: str = None,
-        search_text: str = None,
     ) -> List[str]:
         """
-        Returns a list of tables in a Glue database.
+        Returns a list of tables in Amazon Redshift Spectrum database.
```

Comment:
It was correct originally -- this is a Glue database, as per `awswrangler` docs. Spectrum reads data from Glue catalog.

Reply:
Indirectly it's also Amazon Redshift so I believe we could call it `Amazon Redshift Spectrum` so that it's more transparent for users. They're usually unaware of using Glue. WDYT?

Reply:
Hm in fact both are incorrect, as `database` is actually any external database that user wants to query via Spectrum (it can be Athena, Hive, or Glue, or multiple instances of each etc.). For example, you could have `hive1` and `hive2` databases in Spectrum, which are both Hive databases. 

How about "Returns a list of tables in a specified external database of Amazon Redshift Spectrum."?